.github/workflows/playwright.yml

```diff
@@ -0,0 +1,23 @@
+name: Playwright Tests
+on: [push, pull_request]
+jobs:
+  test:
+    timeout-minutes: 5
+    runs-on: ubuntu-latest
```

Comment:
Do you know if this machine costs any different?

https://github.com/yext/analytics/pull/78/files/43fe152ffcf9a9768f99d1c39b764777f6805c56#r1326142457

Reply:
That's what it was defaulted as, and that's what deploy_hold.yml uses. For clarity, what do you mean by costs?

Reply:
Cool, was just curious.